src/eval-frame/components/panes/ErrorPane.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+
+class ErrorPane extends Component {
```

Comment:
This indentation isn't correct -- the content of the class should be indented two spaces in. Again not sure why our linter isn't picking this up. I'll investigate

Reply:
Couldn't quite figure out why it was passing, but I filed #1199 which will hopefully put some of the ambiguities around accepted coding style to rest for good.